### ISSUE TITLE
Store scrollback next to logs

### ISSUE THREAD

**Issue by jtrakk** [hexchat/hexchat#2455]
The scrollback directory is stored in the hexchat config directory, not in the log directory. I would like the scrollback to have the same parent as the logs, so if I change the log directory, the scrollback directory would also change.

**Comment by TingPing** [comment]
They aren't really files meant to be exposed to the user at all.

**Comment by thedaveCA** [comment]
I agree, because I like to keep my configuration separate from my communications, and to me, the scrollback is communication, despite the fact that I don't personally access those files. I'd be just as happy making it configurable in which case I would put scrollback data into a temp directory that is not backed up at all.

Why? I keep snapshotted backups of my configuration/settings relatively indefinitely, but I need to keep all personally identifying information out. My backups for communications and personal data are in a separate backup set snapshots that are transient in nature, such that if I need to remove something from a log or an entire log, it will age out of all backups in a relatively short period of time.

While my habit of keeping things separate goes back a long way, data hygiene is especially important in today's world where the GDPR imposes significant penalties for losing PII, and where border crossings offer no rights or protection against searches, seizures, and retention of data.
